### ISSUE TITLE
Some questions

### ISSUE THREAD

**Issue by jiaxiaojunQAQ** [val-iisc/NuAT#4]
The used ResNet is different from the previous work, it has an extra layer ReLU?
Whether the results reported in the paper are optimal or final checkpoints?

**Comment by SravantiAddepalli** [comment]
Hi @jiaxiaojunQAQ,

We used the ResNet-18 architecture from the following repository:
https://github.com/yichuan9527/pytorch-cifar/blob/master/models/resnet.py
Yes, there is an additional ReLU layer compared to the standard ResNet-18 model. However, we do not expect the results to be significantly different with the standard model as well. 

We select the best model based on I-FGSM 7-step accuracy on a hold-out validation set. The code for selection of best model is also present in the same file that is used for training models.
